css/ot_pull_to_refresh.css

```diff
@@ -0,0 +1,109 @@
+@-webkit-keyframes ot-spinner-12 {
+    0% { -webkit-transform: rotate(0deg);}
+    8.33% {-webkit-transform: rotate(30deg);}
+    16.66% {-webkit-transform: rotate(60deg);}
+    25% { -webkit-transform: rotate(90deg);}
+    33.33% {-webkit-transform: rotate(120deg);}
+    41.66% {-webkit-transform: rotate(150deg);}
+    50% { -webkit-transform: rotate(180deg);}
+    58.33% {-webkit-transform: rotate(210deg);}
+    66.66% {-webkit-transform: rotate(240deg);}
+    75% {-webkit-transform: rotate(270deg);}
+    83.33% {-webkit-transform: rotate(300deg);}
+    91.66% {-webkit-transform: rotate(330deg);}
+    100% {-webkit-transform: rotate(360deg);}
+}
+
+.ot-pull-to-refresh-wrapper{
+    overflow: hidden ;
+}
+
+.ot-default-head-container {
+    margin-top: -30px;
+    height: 30px;
+    text-align: center;
+    line-height: 30px;
+    font-size: 14px;
+  }
+
+.ot-head-container .ot-display-none{
+    display: none !important;
```

Comment:
Maybe you can try to avoid the useage of `!important` as it is difficult to override.